Gemfile

```diff
@@ -26,11 +26,17 @@ gem 'puma', '~> 3.11'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
+gem 'rack-cors'
```

Comment:
It would be nice to define a gem version to avoid inconsistencies between dev machines, [read more here](http://railsapps.github.io/managing-rails-versions-gems.html)